packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -209,6 +209,8 @@ export const messages = {
   force_apex_log_get_no_logs_text: 'No Apex debug logs were found',
   force_apex_log_get_pick_log_text: 'Pick an Apex debug log to get',
   force_apex_log_list_text: 'Getting Apex debug logs',
+  force_apex_log_get_library: 'Apex Library: Get Logs',
```

Comment:
No customer facing message should have a reference to a change in the implementation (library vs cli). This should be changed to `Get Apex Logs` or `SFDX: Get Apex Logs`.

Reply:
Yeah this message is only used for telemetry, the customer facing message is `Get Apex Logs`

Reply:
Yeah customer facing message is "apex_log_get_text: 'Get Apex Logs' " !

Reply:
@smitshah2531 and @AnanyaJha - do you have everything you need from me?

Reply:
@ekapner I guess yes. This was the only one thing I guess we were worried about. From my side everything looks good to me. Do you have anything @AnanyaJha ?

Reply:
Yes, that was the only customer facing text. thanks for the review emily!